spec/dummy/.controlplane/templates/secrets.yml

```diff
@@ -1,3 +1,8 @@
+kind: secret
```

Comment:
Decided to keep it named `secrets.yml` instead of `org.yml` because:

1. `org.yml` would not create an org, so it would be misleading
2. You could apply `secrets.yml` multiple times if using different secrets for different apps (for example, different secrets for staging and review apps in the same org)

Reply:
I disagree with point "1." Yes, we don't create an org, but this is a one time thing for the org.

I don't understand point number two. Maybe you're trying to say in case you have multiple orgs, then you need to apply this to multiple orgs?

In any case, I'd prefer if this is org.yml. Yes, the org is set up in the UX. So what? After that, you apply the org.yml file. I found the top part 

https://github.com/shakacode/heroku-to-control-plane/pull/159/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R200-R201

```
    # If you're going to use secrets, you need to apply the `secrets.yml` template separately (one-time setup):
    # `cpl apply-template secrets -a my-app`
``

very confusing...

```
    # If you're going to use secrets, you need to apply the `org.yml` template (one-time setup for your org):
    # Pick any app in the relevant org.
    # `cpl apply-template org -a my-app`
```

CC: @dzirtusss

Reply:
Sergey says this is not for staging and production

Reply:
OK -- I get this now. the app name is used to create the right secrets...

and if you specify the command multiple times with different app names, then different secrets get created.

I suggest that we automatically create the org secrets when `setup-app` is used and the secrets and policy don't exist (unless `--skip-secret-access-binding`) is used. 

```
kind: secret
name: {{APP_SECRETS}}
type: dictionary
data: {}
---
# Policy is needed to allow identities to access secrets
kind: policy
name: {{APP_SECRETS_POLICY}}
targetKind: secret
targetLinks:
  - //secret/{{APP_SECRETS}}
```

Reply:
We can do this in a follow up PR.